cvat-ui/src/reducers/index.ts

```diff
@@ -694,6 +694,12 @@ export enum NavigationType {
     EMPTY = 'empty',
 }
 
+export interface EditingState {
+    shapeType: ShapeType | null;
+    editedStateInstance: ObjectState | null;
+    editedStateHidden: boolean;
```

Comment:
I will propose a little renaming, like: 
```suggestion
    objectState: ObjectState | null;
    isHidden: boolean;
```

Reply:
Renaming to `isHidden` is not relevant considering my general comment on review